### ISSUE TITLE
Weird results for tracks listed at album level

### ISSUE THREAD

**Issue by matkoniecz** [RustyRin/bandcamp-api#6]
https://lesfriction.bandcamp.com/music lists some tracks at album level what results in interesting issues:

```
from bandcamp_api import Bandcamp

bc = Bandcamp()
data = bc.get_artist(artist_url = "https://lesfriction.bandcamp.com/album/les-friction")
print('https://lesfriction.bandcamp.com/track/who-will-save-you-now-instrumental' in data.album_urls)
print({'artist': data.artist_title, 'link': data.artist_url, 'album_urls': data.album_urls})
album = bc.get_album(album_url='https://lesfriction.bandcamp.com/track/who-will-save-you-now-instrumental')
print(album.tracks)
print(album.tracks[0].price)
print(album.price)
```
`https://lesfriction.bandcamp.com/track/who-will-save-you-now-instrumental` is in `data.album_urls`

Its data can be fetched with `bc.get_album` - but `album.tracks[0].price` gives price of the entire discography

And `album.price` is `None`

**Comment by matkoniecz** [comment]
Not sure how it should be fixed - add `data.track_urls`? Allow parsing tracks as albums? List album-level tracks in `data.album_urls` but throw exception on passing them to `get_album`, and user should on their own fetch track data for them?

**Comment by RustyRin** [comment]
I am actually already remaking the track objects currently!  
Should be fixed when I finish the rework

**Comment by matkoniecz** [comment]
oh, in such case I guess that making PR would not be a good idea

**Comment by RustyRin** [comment]
Yeah, its a near total rework

**Comment by RustyRin** [comment]
I do think this is now fixed in 0.2.1
I will update the docs in a day or two

**Comment by RustyRin** [comment]
Closed!
